Gems/PhysX/Code/Editor/ComboBoxEditButtonPair.cpp

```diff
@@ -29,6 +29,25 @@ namespace PhysX
 
             layout->addWidget(m_comboBox);
             layout->addWidget(m_editButton);
+
+            connect(m_comboBox, SIGNAL(currentIndexChanged(int)), this, SLOT(onChildComboBoxValueChange(int)));
+        }
+
+        bool ComboBoxEditButtonPair::value() const 
+        {
+            return m_comboBox->currentIndex() == 1;
+        }
+
+        void ComboBoxEditButtonPair::setValue(bool value) 
+        {
+            m_comboBox->blockSignals(true);
+            m_comboBox->setCurrentIndex(value ? 1 : 0);
+            m_comboBox->blockSignals(false);
+        }
+
+        void ComboBoxEditButtonPair::onChildComboBoxValueChange(int value)
+        {
+            emit valueChanged(value == 0 ? false : true);
```

Comment:
This logic is referring to the value, but it looks like it's actually operating on the index instead. Should either be made more generic to handle any value, or should be changed to `CurrentIndex()` / `SetCurrentIndex()`

Reply:
+1 to @moraaar's comment - could you clarify what you mean @cgalvan? Do the `CurrentIndex/SetCurrentIndex` calls already exist?

Reply:
What I mean is that this logic is assuming the combo box will only ever have 2 values, so it's a lot of extra boilerplate for this class `ComboBoxEditButtonPair` where we could instead just add support to the generic combo box handler for an edit attribute (I mentioned this in another comment as well).